### ISSUE TITLE
Discrepancy Between TF Binding Scores and Chromatin Accessibility

### ISSUE THREAD

**Issue by HammadFarooq** [buenrostrolab/scPrinter#46]
Hello,

I’ve been following the tutorial using the PBMC bulk ATAC-seq data, and I was able to successfully reproduce the TF binding scores (TFBs) for each sample in the tutorial using the provided regions.

Building on that, I trained separate seq2PRINT models for each of my bulk ATAC-seq samples, following the same approach. However, I encountered some unexpected results when generating TFBs for one of my regions of interest.

The attached figure shows ATAC-seq signal (left panel) and corresponding TFBs (right panel) across 15 samples for the same 800bp region. Some observations:

- ESC_d1_CEBPA and ESC_d2_hCEBPA show low accessibility, yet have relatively high TFBs.
- Conversely, NSC_d2_OSKM_mCEBPB is highly accessible, but exhibits very low TFBs.

These observations appear inconsistent with what I would expect based on the accessibility data. I'm wondering:

- How should I interpret these results?
- Could this be a modeling artifact or a potential issue in the way the workflow was applied?

Any insights or suggestions would be greatly appreciated!

Thanks,

<img width="1765" alt="Image" src="https://github.com/user-attachments/assets/0647e873-4dfb-4578-b994-e20ea73fe1e7" />

**Comment by ruochiz** [comment]
For the left track, the accessibility, is it normalized? 

But yeah, overall comparing TFBS across condition is something tricky, we have tried to use quantile normalization within each sample across all CRES as one normalization strategy. 

In the paper, all comparisons are made within the same scATAC-seq sample with LoRA finetuning and we ensure the coverage within each pseudobulk / condition is roughly the same to minimize potential bias
